migration.go

```diff
@@ -129,21 +129,23 @@ func (m *Migration) Buffer() error {
 
 	// start reading from body, peek won't move the read pointer though
 	// poor man's solution?
-	if _, err := b.Peek(int(m.BufferSize)); err != nil && err != io.EOF {
+	// related issue : https://github.com/gobuffalo/packd/issues/9
+	migrFile, err := b.Peek(int(m.BufferSize))
```

Comment:
Rename `migrFile` to `peeked`, `peekedData`, or `peekedBytes` since this `[]byte` may contain the full file.